app/src/main/java/com/kks/nimblesurveyjetpackcompose/repo/survey/SurveyRepo.kt

```diff
@@ -0,0 +1,9 @@
+package com.kks.nimblesurveyjetpackcompose.repo.survey
+
+import com.kks.nimblesurveyjetpackcompose.model.ResourceState
+import com.kks.nimblesurveyjetpackcompose.model.response.IncludedResponse
+import kotlinx.coroutines.flow.Flow
+
+interface SurveyRepo {
+    fun getSurveyDetail(surveyId: String): Flow<ResourceState<List<IncludedResponse>>>
```

Comment:
`List`s are usually plural 😉 
```suggestion
    fun getSurveyDetails(surveyId: String): Flow<ResourceState<List<IncludedResponse>>>
```